openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1042,9 +1042,11 @@ Cache-Control: no-store
 
 ### Credential Issuer Provided Nonce {#issuer-provided-nonce}
 
-The Credential Issuer that requires the Client to send a key proof of possession of the key material for the Credential to be bound to (`proof`) MAY receive a Credential Request without such a key proof or with an invalid key proof. In such a case, the Credential Issuer MUST provide the Client with a `c_nonce` defined in (#credential-response) in a Credential Error Response using `invalid_proof` error code defined in (#credential-error-response). 
+The Credential Issuer MAY provide the Client with a `c_nonce` as defined in (#credential-response) in a Credential Error Response using `invalid_proof` error code defined in (#credential-error-response) if the Credential Issuer Metadata contains `proof_types_supported` indicating a key proof is required for the requested Credential. Depending on the Credential Issuer policy, this occurs if they receive a Credential Request without a `c_nonce` or with an invalid server-provided `c_nonce` value included in the proof(s) in the `proof` or `proofs` parameter.
 
-The `c_nonce` value MUST be incorporated in the respective parameter in the `proof` object.
+If the Client has not received a `c_nonce` and the Credential Issuer Metadata contains `proof_types_supported` indicating a key proof is required for the requested Credential, the Client MUST send a Credential Request that contains a `proof` or `proofs` parameter that is fully valid but does not include a server-provided `c_nonce` value. It is the Credential Issuer policy whether or not a `c_nonce` value is required in the key proofs.
```

Comment:
> valid but does not include a server-provided `c_nonce` value.

would this mean that a client can send a proof without a c_nonce and with a c_nonce but that the server should ignore it?

Reply:
> contains a `proof` or `proofs` parameter that is fully valid but does not include a server-provided `c_nonce` value.

This means, the client can send a proof with (if c_nonce was provided) or without (if no c_nonce was provided) a c_nonce. A server would not issue a c_nonce if they would ignore that later, right? Are you saying, we should add a sentence that says that the server has to validate the c_nonce in case a c_nonce was provided and check whether it is valid and matches their credential issuer policy (this can depend on the issuer how to deal c_nonce validation).

Reply:
Updated the PR.